Changes

```diff
@@ -1,6 +1,8 @@
 Change history for libwww-perl
 
-{{$NEXT}}
```

Comment:
This is actually a template variable that `Dist::Zilla` will be looking for.  Lines 4-5 are what we need here.

Reply:
Thanks, that was silly of me. Fixed.